src/pages/tracked-contracts/TrackedContract.tsx

```diff
@@ -1,59 +1,82 @@
 import React, { useEffect, useState } from 'react';
 import { FaCheck, FaTimes, FaTrash } from 'react-icons/fa';
+import ContractsApi from '../../api/contracts';
```

Comment:
I've done it a slightly differently from you @Frenkiee , I like that format of just importing as named import and then calling the function.

The then / catch & everything else is done directly from the component.